lms/templates/discussion/index.html

```diff
@@ -35,9 +35,9 @@
          data-user-cohort-id="${user_cohort}"
          data-course-settings="${course_settings}">
     <div class="discussion-body">
-        <div class="forum-nav"></div>
-        <div class="discussion-column">
-          <article class="new-post-article" style="display: none"></article>
+        <div class="forum-nav" role="complimentary" aria-label="Discussion thread list"></div>
+        <div class="discussion-column" role="complementary" aria-label="Active Discussion" aria-live="assertive" id="discussion-column">
```

Comment:
not sure what the aria-live=assertive is doing here.  this is very dangerous and it will interrupt the screen reader and read any text change in here.    When we talked about this offline, I mentioned putting the aria-live on the "Loading" span, but that is it, and only if we also included a "Loaded" state.